### PR TITLE
[WIP] Jazz template log issue (defect)

### PR DESCRIPTION
**Pull request by ajay1kumar1**
### Requirements
Logs have additional characters in Jazz UI.

### Description of the Change

Defect : log message contains extra chars, need to remove and make it a clean message.

### CODE REVIEW

File: templates/api-template-nodejs/components/logger.js [hunk 1]
```diff
@@ -89,16 +89,14 @@ module.exports = function () {
...
         const timestamp = new Date().toISOString();
-
         const logLevelMessageTypes = {
-            'error': `${timestamp}, 'ERROR \t', ${config.requestDetails}, ${message}`,
-            'warn': `${timestamp}, 'WARN \t', ${config.requestDetails}, ${message}`,
-            'info': `${timestamp}, 'INFO \t', ${config.requestDetails}, ${message}`,
-            'verbose': `${timestamp}, 'VERBOSE \t', ${config.requestDetails}, ${message}`,
-            'debug': `${timestamp}, 'DEBUG \t', ${config.requestDetails}, ${message}`,
-            'log': `${timestamp}, ${level} \t', ${config.requestDetails}, ${message}`
+            'error': ` ${message}`,
```
Comment (svidhani) [inline templates/api-template-nodejs/components/logger.js]: you dont need ` and $. just say message

Comment (ajay1kumar1) [reply templates/api-template-nodejs/components/logger.js]: updated.

File: templates/api-template-nodejs/components/logger.js [hunk 2]
```diff
@@ -89,16 +89,14 @@ module.exports = function () {
 
     const log = (level, message) => {
         const timestamp = new Date().toISOString();
```
Comment (pspani) [inline templates/api-template-nodejs/components/logger.js]: remove timestamp variable as it is no longer used

File: templates/api-template-nodejs/components/logger.js [hunk 3]
```diff
@@ -87,18 +87,15 @@ module.exports = function () {
...
-
+    const log = (level, message) => {        
         const logLevelMessageTypes = {
-            'error': `${timestamp}, 'ERROR \t', ${config.requestDetails}, ${message}`,
-            'warn': `${timestamp}, 'WARN \t', ${config.requestDetails}, ${message}`,
-            'info': `${timestamp}, 'INFO \t', ${config.requestDetails}, ${message}`,
-            'verbose': `${timestamp}, 'VERBOSE \t', ${config.requestDetails}, ${message}`,
-            'debug': `${timestamp}, 'DEBUG \t', ${config.requestDetails}, ${message}`,
-            'log': `${timestamp}, ${level} \t', ${config.requestDetails}, ${message}`
+            'error': message,
```
Comment (svidhani) [inline templates/api-template-nodejs/components/logger.js]: sorry, you need to update in all the templates and remove the comas after reverting the change

Comment (devsatishm) [reply templates/api-template-nodejs/components/logger.js]: Is this not going to break the `log level` parsing logic in the log streamers?

File: templates/api-template-nodejs/components/logger.js (hunk 2 above)
Comment (ajay1kumar1) [reply templates/api-template-nodejs/components/logger.js]: another format updated.

File: templates/api-template-nodejs/components/logger.js (hunk 3 above)
Comment (ajay1kumar1) [reply templates/api-template-nodejs/components/logger.js]: testing for node template. once it showing in correct format then will replicate the same on all 6 templates.

Comment (ajay1kumar1) [reply templates/api-template-nodejs/components/logger.js]: Updated as format of log streamers.

**Pull request merged**